ui/src/app/completed/controllers/TableCtrl.js

```diff
@@ -6,20 +6,50 @@
 
   function TableCtrl($scope, ToastService, TableService, UtilityService) {
     $scope.downloads = [];
-
     var setSize = function(lst) {
       lst.data.forEach(function(download) {
         download.size = UtilityService.formatBytes(download.size);
       })
       return lst;
     };
-
+    $scope.isGridOn = false;
     TableService.getCompletedDownloads().then(function (response) {
       response = setSize(response);
       $scope.downloads = response.data;
     }, function(error){
       ToastService.showToast('Oops! Something went wrong fetching data');
     });
+    $scope.changeView = function () {
+        if($scope.isGridOn) {
+            document.getElementById('grid-button').innerText = 'view_module';
+            $scope.isGridOn = false;
+        }else{
+            document.getElementById('grid-button').innerText = 'view_list';
+            $scope.isGridOn = true;
+        }
+    };
+    $scope.getImage = function (fileName) {
+        var fileExtensionName = getExtension(fileName);
```

Comment:
inconsistent indentations.

Reply:
ok, so as long I can dive to the code mean I go through files, I will change indentations accordingly. Or should I do all these in last weeks?